packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/setup.dev.flow.ts

```diff
@@ -0,0 +1,91 @@
+import { FlowCommand } from '@chainlink/gauntlet-core'
+import { CATEGORIES } from '../../../lib/constants'
+import { waitExecute, TransactionResponse } from '@chainlink/gauntlet-terra'
+import { makeAbstractCommand } from '../../abstract'
+import UploadContractCode from '../../tooling/upload'
+import DeployLink from '../link/deploy'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import DeployOCR2 from './deploy'
+import SetBilling from './setBilling'
+import SetConfig from './setConfig'
+import SetPayees from './setPayees'
+
+export default class DeployOCR2Flow extends FlowCommand<TransactionResponse> {
+  static id = 'ocr2:setup:flow'
+  static category = CATEGORIES.OCR
+
+  // yarn gauntlet ocr2:deploy:flow --network=bombay-testnet --decimals=8 --description="OCR2" --max_answer=10000000 --min_answer=0
+
+  constructor(flags, args) {
+    super(flags, args, waitExecute)
+
+    this.stepIds = {
+      BILLING_ACCESS_CONTROLLER: 1,
+      REQUEST_ACCESS_CONTROLLER: 2,
+      LINK: 3,
+      OCR_2: 4,
+    }
+
+    this.flow = [
+      // {
+      //   name: 'Upload Contracts',
+      //   command: UploadContractCode,
+      // },
+      // {
+      //   name: 'Deploy LINK',
+      //   command: DeployLink,
+      //   id: this.stepIds.LINK,
+      // },
+      // {
+      //   name: 'Deploy Billing Access Controller',
+      //   command: 'access_controller:deploy',
+      //   id: this.stepIds.BILLING_ACCESS_CONTROLLER,
+      // },
+      // {
+      //   name: 'Deploy Request Access Controller',
+      //   command: 'access_controller:deploy',
+      //   id: this.stepIds.REQUEST_ACCESS_CONTROLLER,
+      // },
```

Comment:
This should probably be excluded from the ocr2 deploy flow as we often want to reuse LINK & AC contracts, right?

Reply:
True